app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+
+/*= require uikit */
```

Comment:
Szinten ezt is elorebb tennem, mert igy nem lesz egyszeru felulirni a UIKites sass valtozokat vagy barmit, mert a `require_tree .` es a `require_self` elobb van. Sot a sass miatt erdemes ezt atalakitani ugy hogy a sass `@import` statementet hasznalja a `require` directive helyett. 

A bootstrap is hasonlo: https://github.com/twbs/bootstrap-sass#a-ruby-on-rails